src/en/manhwalike/src/eu/kanade/tachiyomi/extension/en/manhwalike/Manhwalike.kt

```diff
@@ -0,0 +1,223 @@
+
+    override fun latestUpdatesSelector() = "ul.slick_item div.visual"
+
+    override fun latestUpdatesNextPageSelector() = popularMangaNextPageSelector()
+
+    override fun latestUpdatesFromElement(element: Element): SManga = popularMangaFromElement(element)
+
+    // search
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        val filterList = filters.let { if (it.isEmpty()) getFilterList() else it }
+        return if (query.isNotEmpty()) {
+            val requestBody = query.toFormRequestBody()
+            val requestHeaders = headersBuilder().buildApiHeaders(requestBody)
+            POST("$baseUrl/search/html/1", requestHeaders, requestBody)
+        } else {
+            val url = baseUrl.toHttpUrl().newBuilder()
+            filterList.forEach { filter ->
+                when (filter) {
+                    is GenreFilter -> filter.toUriPart().let { url.addPathSegment(it) }
+                    else -> {}
+                }
+            }
+            url.addQueryParameter("page", page.toString())
+            return GET(url.build(), headers)
+        }
+    }
+
+    override fun searchMangaSelector() = "ul.normal li"
+
+    override fun searchMangaNextPageSelector() = "ul.pagination li:last-child a"
+
+    override fun searchMangaParse(response: Response): MangasPage {
+        val document = response.asJsoup()
+        val mangas = when {
+            document.select(searchMangaSelector()).isEmpty() -> document.select("ul li").map { element ->
+                searchMangaFromElement(element)
+            }
+            else -> document.select(searchMangaSelector()).map { element ->
+                searchMangaFromElement(element)
+            }
+        }
+        val hasNextPage = searchMangaNextPageSelector().let { selector ->
+            document.select(selector).first()
+        } != null
+        return MangasPage(mangas, hasNextPage)
+    }
+
+    override fun searchMangaFromElement(element: Element): SManga {
+        return SManga.create().apply {
+            title = element.selectFirst("img")!!.attr("alt")
+            thumbnail_url = when {
+                element.selectFirst("img")!!.hasAttr("data-original") -> element.selectFirst("img")!!.attr("data-original")
+                else -> element.selectFirst("img")!!.attr("src")
+            }
+            setUrlWithoutDomain(element.selectFirst("a")!!.attr("href"))
+        }
+    }
+
+    // details
+    override fun mangaDetailsParse(document: Document): SManga {
+        return SManga.create().apply {
+            author = document.selectFirst("div.author a")!!.text()
+            status = document.selectFirst("small:contains(Status) + strong")!!.text().trim().toStatus()
+            genre = document.select("div.categories a").joinToString { it.text() }
+            description = document.select("div.summary-block p.about").text()
+            thumbnail_url = document.select("div.fixed-img img").attr("src")
```

Comment:
```suggestion
            thumbnail_url = document.select("div.fixed-img img").attr("abs:src")
```

Reply:
Should use `.absUrl()`, but that only works for `Element` not `Elements`. You can get `Element` with `.selectFirst()` (or by iteration, not relevant here)